src/entry.js

```diff
@@ -1,11 +1,13 @@
-const shared = require('./shared');
+const shared = require('./utils/shared');
 
 function StacheEntryPlugin() {
   const preload = (content, resourcePath, skyPagesConfig) => {
     try {
       const preloadPluginOrder = [
         require('./config'),
+        require('./json-data-element-attributes'),
         require('./include'),
+        require('./json-data-build-time'),
         require('./code-block'),
         require('./json-data'),
         require('./route-metadata'),
```

Comment:
Mind talking me through the order here? It seems that we'd want `json-data` to run first (to generate the JSON object), then do a blanket run of `json-data-build-time`, then parse any attributes `json-data-element-attributes`. I'd also be assuming that these would be running before any other plugins (since they're modifying the actual source code), just to be safe (I can see the use-case of having `config` run at the very top of the list).

I could be missing something critical, however!